stanzaService.py

```diff
@@ -128,15 +128,16 @@ def map_entity(self, e):
 
     @staticmethod
     def offset_id(t):
-        return "-".join(map(str, [t.start_char, t.end_char]))
+        return f"{t.start_char}-{t.end_char}"
 
     # The ID of a token is built out of the index of the token in the sentence
     # (a tupel as this reports multi-word tokens on the same index with a sub
     # index for sub-tokens) as well as the start/end offset if the token
     def token_id(self, t):
-        return ".".join(["-".join(map(str, t.id)), self.offset_id(t)])
+        index_id = "-".join([str(index) for index in t.id])
```

Comment:
Can there ever be more than two indexes in the `t.id`? If not, this would basically collapse onto

```
return f"{t.id[0]}-{t.id[1]}.{self.offset_id(t)}"
```

🤞

Reply:
in 99.9% of cases `t.id` has only a single element. So `t.id[1]` does not exist. I have no idea if there is a possibility of > 2 elements.